[ 2025-02-03 14:23:08,390 ] matplotlib - DEBUG - matplotlib data path: /opt/anaconda3/envs/yt/lib/python3.11/site-packages/matplotlib/mpl-data
[ 2025-02-03 14:23:08,408 ] matplotlib - DEBUG - CONFIGDIR=/Users/dhruvshrinet/.matplotlib
[ 2025-02-03 14:23:08,409 ] matplotlib - DEBUG - interactive is False
[ 2025-02-03 14:23:08,409 ] matplotlib - DEBUG - platform is darwin
[ 2025-02-03 14:23:08,431 ] matplotlib - DEBUG - CACHEDIR=/Users/dhruvshrinet/.matplotlib
[ 2025-02-03 14:23:08,433 ] matplotlib.font_manager - DEBUG - Using fontManager instance from /Users/dhruvshrinet/.matplotlib/fontlist-v390.json
[ 2025-02-03 14:23:09,847 ] tensorflow - DEBUG - Falling back to TensorFlow client; we recommended you install the Cloud TPU client directly with pip install cloud-tpu-client.
[ 2025-02-03 14:23:10,033 ] h5py._conv - DEBUG - Creating converter from 7 to 5
[ 2025-02-03 14:23:10,033 ] h5py._conv - DEBUG - Creating converter from 5 to 7
[ 2025-02-03 14:23:10,033 ] h5py._conv - DEBUG - Creating converter from 7 to 5
[ 2025-02-03 14:23:10,033 ] h5py._conv - DEBUG - Creating converter from 5 to 7
[ 2025-02-03 14:23:10,541 ] root - INFO - Data ingestion started.
[ 2025-02-03 14:23:10,541 ] root - INFO - Entered the data ingestion method or component
[ 2025-02-03 14:23:10,566 ] root - INFO - Read the dataset as dataframe
[ 2025-02-03 14:23:10,566 ] root - INFO - Train test split initiated
[ 2025-02-03 14:23:10,617 ] root - INFO - Inmgestion of the data iss completed
[ 2025-02-03 14:23:10,617 ] root - INFO - Data ingestion completed.
[ 2025-02-03 14:23:10,617 ] root - INFO - Data ingestion artifacts: ('artifacts/train.csv', 'artifacts/test.csv')
[ 2025-02-03 14:23:10,617 ] root - INFO - Entered the start_data_transformation method of TrainPipeline class
[ 2025-02-03 14:23:10,617 ] root - INFO - Data transformation started.
[ 2025-02-03 14:23:11,567 ] root - INFO - Data preprocessing completed.
[ 2025-02-03 14:23:11,590 ] root - ERROR - Error occurred during pipeline execution: Error occurred python script name [/Users/dhruvshrinet/Desktop/NLP/youtubeComments/pipeline/train_pipeline.py] line number [33] error message [DataTransformationArtifacts.__init__() got an unexpected keyword argument 'index']
